ext/net/Cargo.toml

```diff
@@ -17,8 +17,9 @@ path = "lib.rs"
 deno_core = { version = "0.140.0", path = "../../core" }
 deno_tls = { version = "0.45.0", path = "../tls" }
 log = "0.4.16"
+openssl = { version = "0.10", features = ["vendored"] }
 serde = { version = "1.0.136", features = ["derive"] }
 socket2 = "0.4.4"
 tokio = { version = "1.17", features = ["full"] }
-trust-dns-proto = "=0.21.2"
+trust-dns-proto = { version = "=0.21.2" }
```

Comment:
It looks like this change is no longer necessary.

Reply:
I've forgetten to remove from main cargo.toml, sorry.